packages/manager/modules/exchange/src/account/update/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,85 @@
+  "exchange_account_update_password_hint_complex_1_3": "doit comporter les éléments de trois des quatre catégories suivantes :",
+  "exchange_account_update_password_hint_complex_1_3_1": "des lettres minuscules,",
+  "exchange_account_update_password_hint_complex_1_3_2": "des lettres majuscules,",
+  "exchange_account_update_password_hint_complex_1_3_3": "des chiffres entre 0 et 9,",
+  "exchange_account_update_password_hint_complex_1_3_4": "des caractères spéciaux.",
+  "exchange_account_update_passwordConfirmation_label": "Confirmation",
+  "exchange_account_update_confirm_button": "Enregistrer les modifications",
+  "exchange_account_update_cancel_button": "Annuler",
+  "exchange_account_update_field_error_required": "Veuillez compléter ce champ.",
+  "exchange_account_update_field_error_doesntRespectComplexityRules": "Ce champ ne respecte pas les règles de complexité.",
+  "exchange_account_update_field_error_passwordConfirmation": "Mots de passe différents.",
+  "exchange_account_update_field_error_pattern": "Veuillez saisir un format valide.",
+  "exchange_account_update_emailAddress_error_alreadyExists": "Un compte avec cette adresse e-mail existe déjà.",
+  "exchange_account_update_submit_success": "La création du compte {0} a été effectuée. Vous pourrez l’utiliser dans quelques minutes.",
+  "exchange_ACTION_update_account_legal_title": "J'accepte les conditions d'utilisation",
+  "exchange_ACTION_update_account_legal_warning": "Je reconnais qu'OVH procédera à l’exécution immédiate de la prestation à compter de la validation de ma commande et à ce titre je renonce expressément à exercer mon droit de rétractation conformément aux dispositions de l’article L.121-21-8 du code de la consommation.",
+  "exchange_ACTION_update_account_title": "Modifier le compte Exchange",
+  "exchange_ACTION_update_account_step1_email_label": "Compte e-mail",
+  "exchange_ACTION_update_account_button": "Modifier le compte",
+  "exchange_ACTION_update_account_step1_first_name_label": "Prénom",
+  "exchange_ACTION_update_account_step1_last_name_label": "Nom",
+  "exchange_ACTION_update_account_step1_display_name_label": "Nom à afficher",
+  "exchange_ACTION_update_account_step2_confirmation_intro": "Votre compte :",
+  "exchange_ACTION_update_account_step2_email_label": "Compte e-mail",
+  "exchange_ACTION_update_account_step2_first_name_label": "Prénom",
+  "exchange_ACTION_update_account_step2_last_name_label": "Nom",
+  "exchange_ACTION_update_account_step2_display_name_label": "Nom à afficher",
+  "exchange_ACTION_update_account_GAL_true": "Oui",
+  "exchange_ACTION_update_account_GAL_false": "Non",
+  "exchange_ACTION_update_account_outlook_true": "Oui",
+  "exchange_ACTION_update_account_outlook_false": "Sans",
+  "exchange_ACTION_update_no_domains": "Vous n'avez pas encore ajouté de domaine ou il n'a pas encore été validé. <br/>Vous devez d'abord ajouter un domaine et attendre qu'il soit validé pour créer un compte Exchange.",
```

Comment:
**nitpick(non-blocking):** We should split this translation in two parts and get rid of `<br/>` element.